@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -724,6 +731,32 @@ export class TileGeometryCreator {
                     object.userData.geometryId = srcGeometry.uuid;
                 }
 
+                if (isFillTechnique(technique) && (technique.clipping || technique.transparent)) {
```

Comment:
i believe the test should be a bit different
`(clipping || (clipping === undefined && transparent === true)`

Reply:
isn't this the same? Because clipping === undefined is anyway false. Or do you mean I should avoid truthy?

Reply:
no, it's not the same. clipping can be `true`, `false` or `undefined`. if users explicitly sets `clipping` to `false` it doesn't matter if transparent is `true` or `false`, clipping should be disabled, on the other hand if the `clipping` is left `undefined` then we should clip if `transparent` is true.

Reply:
My logic failed me, thank you for explaining.